kernel/kernel-impl/src/test/java/org/sakaiproject/authz/impl/DevolvedSakaiSecurityTest.java

```diff
@@ -79,13 +88,19 @@ public void testCheck() throws Exception {
 		// Make people in site2 with the right permission be able to modify site one.
 		session.setUserId("user1");
 		security.setAdminRealm(site1.getReference(), site2.getReference());
-		
+
+        // TODO This should really invalidate correctly.
```

Comment:
It means admin sites changes don't get picked up until the authz cache times out.